pages/thank-you.vue

```diff
@@ -0,0 +1,326 @@
+<template>
```

Comment:
Could you remove this from this PR? It looks like it (as well as the contents of /assets/thank_you_page/) sneaked in from https://github.com/covidwatchorg/website/pull/155.

Reply:
Yep! My bad, removed from latest commit.